### PR TITLE
fix: produceLKG does not work on Windows

### PR DESCRIPTION
**Pull request by Jack-Works**
I'm publishing my own TypeScript version and I found the build script is not working on Windows.

### CODE REVIEW

File: scripts/produceLKG.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ async function copyFromBuiltLocal(fileName: string) {
 }
 
 async function copyFilesWithGlob(pattern: string) {
-    const files = glob.sync(path.join(source, pattern)).map(f => path.basename(f));
+    const files = glob.sync(path.join(source, pattern).replace(/\\/g, "/")).map(f => path.basename(f));
```
Comment (jakebailey) [inline scripts/produceLKG.ts:93]: Maybe just use `path.posix.join` instead?

Comment (Jack-Works) [reply scripts/produceLKG.ts:93]:
```
> path.posix.join("C:\\Users\\Jack\\", "src/**/*.ts")
'C:\\Users\\Jack\\/src/**/*.ts'
```

It does not work 🤔

Comment (jakebailey) [reply scripts/produceLKG.ts:93]: I see. Maybe the calls to glob shouldn't be messing with the paths at all, and should instead pass the path as the cwd option (then pass the pattern in unchanged).

Comment (jakebailey) [reply scripts/produceLKG.ts:93]: @Jack-Works did you still have time to do this, or would you like me to edit this PR / send a new one?

Comment (Jack-Works) [reply scripts/produceLKG.ts:93]: yes, I can try it

**Review by jakebailey — APPROVED** [review]

**Pull request merged**